arcee/config.py

```diff
@@ -39,10 +39,4 @@ def get_conditional_configuration_variable(key: str, default: str) -> str:
 ARCEE_API_URL = get_conditional_configuration_variable("ARCEE_API_URL", "https://api.arcee.ai")
 ARCEE_APP_URL = get_conditional_configuration_variable("ARCEE_APP_URL", "https://app.arcee.ai")
 ARCEE_API_KEY = get_conditional_configuration_variable("ARCEE_API_KEY", "")
-ARCEE_RETRIEVAL_URL = get_conditional_configuration_variable(
```

Comment:
we will be going through the API now. This isn't secure, and should be handled via `/retrieve` and `/generate` so we get correct permissioning